tests/mentorship_relation/test_api_create_request.py

```diff
@@ -0,0 +1,54 @@
+import unittest
```

Comment:
this test file is already created at `tests/mentorship_relation/test_api_send_request.py`. Can you move the test there, to avoid having tests from the same context in separate files?